x-pack/test/alerting_api_integration/spaces_only/config.ts

```diff
@@ -13,4 +13,5 @@ export default createTestConfig('spaces_only', {
   license: 'trial',
   enableActionsProxy: false,
   rejectUnauthorized: false,
+  preconfiguredAlertHistoryEsIndex: true,
```

Comment:
Do we need a security_and_spaces test to ensure the preconfigured action is **NOT** available?

Reply:
The `get_all` test in `security_and_spaces` will already fail if the preconfigured action shows up in the response.